modules/distribution/resources/migration-1.8.0_to_1.9.0/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom18to19.java

```diff
@@ -0,0 +1,702 @@
+        } catch (UserStoreException e) {
+            ResourceUtil.handleException("Error occurred while reading tenant information", e);
+        }
+        if (log.isDebugEnabled()) {
+            log.debug("Rxt resource migration done for all the tenants");
+        }
+    }
+
+
+
+
+    /**
+     * This method is used to clean old registry resources.
+     * This deletes the old swagger v1.2 resource from the registry
+     *
+     * @throws APIMigrationException
+     */
+    @Override
+    public void cleanOldResources() throws APIMigrationException {
+        log.info("Resource cleanup started for API Manager 1.9.0");
+        try {
+            for (Tenant tenant : tenantsArray) {
+                PrivilegedCarbonContext.startTenantFlow();
+                PrivilegedCarbonContext.getThreadLocalCarbonContext().setTenantDomain(tenant.getDomain());
+                PrivilegedCarbonContext.getThreadLocalCarbonContext().setTenantId(tenant.getId());
+
+                String adminName = ServiceHolder.getRealmService().getTenantUserRealm(
+                        tenant.getId()).getRealmConfiguration().getAdminUserName();
+                ServiceHolder.getTenantRegLoader().loadTenantRegistry(tenant.getId());
+                Registry registry = ServiceHolder.getRegistryService().getGovernanceUserRegistry(adminName,
+                        tenant.getId());
+                GenericArtifactManager manager = new GenericArtifactManager(registry, "api");
+                GovernanceUtils.loadGovernanceArtifacts((UserRegistry) registry);
+                GenericArtifact[] artifacts = manager.getAllGenericArtifacts();
+
+                for (GenericArtifact artifact : artifacts) {
+                    API api;
+
+                    api = APIUtil.getAPI(artifact, registry);
+
+                    APIIdentifier apiIdentifier = api.getId();
+                    String apiName = apiIdentifier.getApiName();
+                    String apiVersion = apiIdentifier.getVersion();
+                    String apiProviderName = apiIdentifier.getProviderName();
+
+                    String swagger12location = ResourceUtil.getSwagger12ResourceLocation(apiName, apiVersion,
+                            apiProviderName);
+
+                    if (registry.resourceExists(swagger12location)) {
+                        registry.delete(APIConstants.API_DOC_LOCATION);
+                        log.info("Resource deleted from the registry.");
+
+                    }
+
+                }
+            }
+        } catch (APIManagementException e) {
+            ResourceUtil.handleException("API Management Exception occurred while migrating rxt.", e);
+        } catch (UserStoreException e) {
+            ResourceUtil.handleException("Error occurred while reading tenant admin.", e);
+        } catch (RegistryException e) {
+            ResourceUtil.handleException("Error occurred while accessing the registry.", e);
+        }
+        if (log.isDebugEnabled()) {
```

Comment:
Debug log changed to info logs in the commit d4354ac636c9ac167ae1f55c7eb428264fce96de